v_3_5_0/master_template.go

```diff
@@ -1440,6 +1442,8 @@ write_files:
       MANIFESTS="${MANIFESTS} default-backend-svc.yml"
       MANIFESTS="${MANIFESTS} ingress-controller-cm.yml"
       MANIFESTS="${MANIFESTS} ingress-controller-dep.yml"
+      {{ end -}}
+      {{ if not .DisableIngressControllerService -}}
```

Comment:
Those indentations look weird. Maybe just put the `if`s at the very beggining of the line. This will give extra visibility what can be disabled.

Reply:
I see what you mean about the indents but these are the same as for the current IC param. I'll test without them and see how it looks.